runtime/hashing/common/test/aws/smithy/kotlin/runtime/hashing/Crc32cTest.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.hashing
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class Crc32cTest {
```

Comment:
Are there any other tests to add here? Maybe take a look at the [CRT tests](https://github.com/awslabs/aws-checksums/blob/main/tests/crc_test.c) to see if there are other test cases we may want to add to this test suite